packages/pagination/src/usePagination.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useSelection } from '@zendeskgarden/container-selection';
+
+/**
+ * A React Hook used to create accessible widgets with the Pagination Interaction Pattern
+ *
+ * https://www.w3.org/TR/wai-aria-practices/#Listbox
+ */
+export function usePagination(options) {
+  const {
+    selectedItem,
+    focusedItem,
+    getContainerProps: getControlledContainerProps,
+    getItemProps
+  } = useSelection(options);
+
+  const getContainerProps = (props = {}) => {
+    return {
+      'aria-label': 'Pagination navigation',
+      ...props
+    };
+  };
+
+  const getPreviousPageProps = (props = {}) => {
+    return {
+      'aria-label': 'Previous Page',
+      ...props
+    };
+  };
+
+  const getNextPageProps = (props = {}) => {
+    return {
+      'aria-label': 'Next Page',
+      ...props
+    };
+  };
+
+  const getPageProps = ({ page, current, ...other } = {}) => {
+    let ariaLabel = `Page ${page}`;
+
+    if (current) {
+      ariaLabel = `Current page, Page ${page}`;
+    }
+
+    return {
+      'aria-label': ariaLabel,
```

Comment:
Shouldn't all the `ariaLabel` usage in this module be exposed as propGetter function parameters to allow for ease of translation?

Reply:
Yep, this was the current behaviour but this makes sense.